src/ts/FileListDialog_Item.html

```diff
@@ -1,6 +1,4 @@
-<ul>
-	<li data-set="this">
+	<li>
 		<input type="radio" name="file-list" data-set="input" />
-		<span data-set="title"></span>
-	</li>
-</ul>
\ No newline at end of file
+		<label data-set="title"></label>
+	</li>
```

Comment:
You should probably include the </ul> for this to be consistent. (I think dtemplate manages this without an issue, but better to have it clear here)

Reply:
Thanks, @craigmj – your code doesn't seem to have come through in your comment here, so I'm not sure what you're referring to. Can you clarify, please?